### PR TITLE
actually adds slugcat to the vendor and donator items because i forgot to hit "push" last time 

### PR DESCRIPTION
**Pull request by thegoldencat413**
# Document the changes in your pull request
i feel like an idiot

this fixes blahaj sprites and makes the slugcat sprites better
also adds them to donator items and to the vendor becuase i messed up last time

# Why is this good for the game?
uhh not really relevant becuase this is a direct copy paste of changes that i didnt push
# Testing
image
image

# Spriting

new:
image
image
image
image

# Wiki Documentation

# Changelog

:cl:  
rscadd: Added slugcat to the vendor and donator items

/:cl:

### CODE REVIEW

**Review by Majkl-J — APPROVED** [review]
Looks good to me

**Comment by Moltijoe** [comment]
update to master

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -600,7 +600,7 @@
 	icon_state = "plush_cdragon"
 	item_state = "plush_cdragon"
 
-/obj/item/toy/plush/catslug
+/obj/item/toy/plush/slugcat
```
Comment (ToasterBiome) [inline code/game/objects/items/plushes.dm:603]: why

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

**Comment by MajManatee** [comment]
Inactive 2+ weeks. Reopen when updated and requests addressed.

**Comment by Maia-J** [comment]
Saying why isn't a request

**Pull request closed without merging**